docs/conf/australia/2022/news/thank-you-recap.rst

```diff
@@ -0,0 +1,118 @@
+Check out the `Write the Docs Slack`_.
+
+.. _Write the Docs Slack: http://www.writethedocs.org/slack/
+
+Write the Docs Newsletter
+-------------------------
+
+Our `monthly newsletter`_ is the other place to keep track of everything that is happening with the community. 
+Subscribe to make sure you hear about all the conferences, meetups, job postings, and summaries of relevant conversations in our Slack.
+
+.. _monthly newsletter: http://writethedocs.org/newsletter/
+
+Local Meetups
+-------------
+
+We hope you continue to `build your local or online communities`_ and keep in touch with other documentarians. 
+We have a number of meetups in cities across the country, and if you don’t see your city listed consider `starting`_ a meetup!
+
+New this year are our `Quorum Meetups`_ which are virtual and targeting large geographic areas. There’s already one covering most everyone in the US or in a similar timezone.
+
+.. _build your local or online communities: http://www.writethedocs.org/meetups/
+.. _starting: http://www.writethedocs.org/organizer-guide/meetups/starting/
+.. _Quorum Meetups: https://www.writethedocs.org/meetups/#current-meetups-quorum
+
+Code of Conduct transparency report
+===================================
+
+As with any Write the Docs event, this conference was covered by our `community Code of Conduct <https://www.writethedocs.org/code-of-conduct/>`__.
+We aim to be as transparent with CoC incidents and enforcement as we can.
+
+To ensure that our CoC was visible and accessible, we took a number of specific steps:
+
+- Every participant agreed to the Code of Conduct in advance during registration.
+- The CoC, or a summary, was included on the conference website and Hopin Reception
+- The CoC was repeatedly mentioned in our conference introduction and announcements.
+- We specifically stressed that the CoC applies to all conference spaces including the social event.
+- We encouraged all attendees to report any incident, even if they were not sure whether it was a violation.
+
+None of our organizers were approached about any CoC concerns at this event.
+We know that this is, in part, a reflection of the thoughtful and inclusive community that we're so lucky to have.
+
+That being said, we're also very conscious of the fact that just because nothing was reported doesn't mean nothing happened.
+If there was any personal conduct or other aspect of the conference experience that did not meet the expectations of behaviour set out in the CoC, please email us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.
+
+Ready for more?
+===============
+
+We are already talking 2023 folks! Our `Portland conference <https://www.writethedocs.org/conf/portland/2023/>`__ has announced its dates for 7-9 May and CFPs are now open.
```

Comment:
This too I feel is kinda heavy for an mostly Australian list?

Reply:
I think this update goes out to everyone WTD, so I'd leave this in, but will take out the Slack/Newsletter/local meetups sections